api/config/config.go

```diff
@@ -336,6 +337,13 @@ type KafkaConfig struct {
 	MaxMessageSizeBytes int    `validate:"required" default:"1048588"`
 	ConnectTimeoutMS    int    `validate:"required" default:"1000"`
 	SerializationFmt    string `validate:"required" default:"protobuf"`
+	LingerMS            int    `validate:"required" default:"0"`
+	Acks                int    `validate:"required" default:"-1"`
```

Comment:
0 Acks will give you the best performance at the expense of slight possibility of losing data when there are connectivity issue between producer <-> Kafka broker. Since losing observation log is less impactful that latency increase, we will mostly use 0. We will also use linger ms > 0 under high RPS scenario so that the Kafka cluster does not get overwhelmed.

So, it depends on whether the expectation is for the Merlin administrators/maintainers to always override the default config for the different environment, or if we should set the default config such that it caters for high RPS use case.